.github/workflows/sub-build.yml

```
@@ -44,6 +41,7 @@ jobs:
     with:
       nodeVersion: ${{ vars.NODE_VERSION }}
       camsServerHostname: ${{ inputs.apiFunctionName }}.azurewebsites.us
+      camsStagingHostname: ${{ inputs.apiFunctionName }}-staging.azurewebsites.us
```

Consider using the variable instead of the hard-coded slot name:
```suggestion
      camsStagingHostname: ${{ inputs.apiFunctionName }}-$${{ vars.SLOT_NAME }}.azurewebsites.us
```